scripts/hooks/commit-msg.sh

```diff
@@ -0,0 +1,72 @@
+#! /bin/bash
+
```

Comment:
We should add comment at the top explaining what this does and how the configuration is set.
```suggestion
# Hook after commit message is set. Reviews the message for a prefix (containing a `:`) that is valid
# Valid prefixes are defined in `./.git-config.types`

```